assets/js/validateFormRegister.js

```diff
@@ -0,0 +1,31 @@
+document.getElementById('formRegister').addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const fullName = document.querySelector('input[name="fullName"]');
+  const phoneNumber = document.querySelector('input[name="phoneNumber"]');
+  const email = document.querySelector('input[name="email"]');
+  const webURL = document.querySelector('input[name="webURL"]');
+  const password = document.querySelector('input[name="password"]');
+  const confirmPassword = document.querySelector('input[name="confirmPassword"]');
+
+  const fullNameError = document.getElementById('fullNameError');
+  const phoneNumberError = document.getElementById('phoneNumberError');
+  const emailError = document.getElementById('emailError');
+  const webURLError = document.getElementById('webURLError');
+  const passwordError = document.getElementById('passwordError');
+  const confirmPasswordError = document.getElementById('confirmPasswordError');
+
+  let isValid = true;
+  const urlPattern = /^(https?:\/\/)?([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,6}(\/[a-zA-Z0-9-._~:/?#[\]@!$&'()*+,;=]*)?$/;
+
+  fullName.value === '' ? (fullNameError.classList.remove('hidden'), isValid = false) : fullNameError.classList.add('hidden');
```

Comment:
```suggestion
  if (fullName.value === '') {
    fullNameError.classList.remove('hidden'); 
    isValid = false
  } else {
    fullNameError.classList.add('hidden')
  }
```
Các thành phần này em dùng if else để code nhìn clean hơn